core/src/main/scala/org/apache/spark/deploy/history/ApplicationHistoryProvider.scala

```diff
@@ -109,4 +109,11 @@ private[history] abstract class ApplicationHistoryProvider {
   @throws(classOf[SparkException])
   def writeEventLogs(appId: String, attemptId: Option[String], zipStream: ZipOutputStream): Unit
 
+  /**
+   * Returns an ApplicationHistoryInfo for the appId.
+   *
+   * @return ApplicationHistoryInfo of one appId if exists.
```

Comment:
This is kind of verbose. I would just say `@return the [[ApplicationHistoryInfo]] for the appId if it exists`